lib/train/platform.rb

```diff
@@ -0,0 +1,80 @@
+# encoding: utf-8
```

Comment:
Should this file not be located in `lib/train/platforms/platform`?

Reply:
Yeah, I do think that should be moved in after thinking about it.